test/common/network/io_socket_handle_impl_test.cc

```diff
@@ -50,6 +61,45 @@ TEST(IoSocketHandleImplTest, TestIoSocketError) {
   EXPECT_EQ(errorDetails(123), error9.getErrorDetails());
 }
 
+#if defined(__linux__)
```

Comment:
This should probably be `TCP_INFO` to match implementation.

Reply:
Done.